### PR TITLE
Offset pages when there are more than 10. closes #3339

### LINKED ISSUE #3339: Rx history pagination when many pages
Mock from Gina:

cdaa1456-9072-11e6-8de3-9b1e800bce3c

The number to show might need to be less than 10 on smaller screens though, use your best judgement there or feel free to ask in this issue for clarification if any edge cases pop up! 

Aside: this will likely be hard to test, as we currently don't have that many prescriptions. Perhaps making the API calls be something like "per_page=2" will help?

### PR DESCRIPTION
**Pull request by webinista**
- Restyling pagination to remove flexbox.
- Moving next/previous links outside of va-pagination-inner.

### CODE REVIEW

**Comment by webinista** [comment]
@crwallace @jbalboni Also tagging the two of you for opinions since this is a common component. See the specs in #3339.

**Comment by jbalboni** [comment]
@webinista This looks great. I wish the design had some sort of indication for the user that there are more pages when you're over 10 pages.

File: src/js/common/components/Pagination.jsx [hunk 1]
```diff
@@ -24,43 +26,50 @@ class Pagination extends React.Component {
...
+  }
 
   render() {
     if (this.props.pages === 1) {
       return <div/>;
     }
 
-    const pageList = Array(this.props.pages).fill().map((e, i) => {
-      const pageNumber = i + 1;
+    const pageList = this.pageNumbers(10).map((e) => {
```
Comment (U-DON) [inline src/js/common/components/Pagination.jsx]: `e` is a pretty non-descriptive name. Can this be `pageNumber` or something now that it actually is the number?

File: src/sass/modules/_va-pagination.scss [hunk 2]
```diff
@@ -5,80 +5,90 @@
...
+    display: inline-block;
+  }
+
+  &-prev {
+    @media (max-width: $medium-screen) {
+      margin-left: -3rem;
+      padding: 0 1rem 0 0;
+    }
+
+    @media (min-width: $medium-screen) {
```
Comment (U-DON) [inline src/sass/modules/_va-pagination.scss]: Can remove this block since it's repeated in line 61.

File: src/sass/modules/_va-pagination.scss [hunk 3]
```diff
@@ -5,80 +5,90 @@
...
+          content: '\a0\a0\003e';
+        }
+      }
+    }
+  }
+
+  &-prev,
+  &-next {    
+    position: relative;
+    top: -1rem;
```
Comment (U-DON) [inline src/sass/modules/_va-pagination.scss]: Can remove these two lines by adding `vertical-align: top` to the `> *` selector.

File: src/sass/modules/_va-pagination.scss [hunk 4]
```diff
@@ -5,80 +5,90 @@
...
 
-  @media (max-width: $small-screen) {
-    @supports (display: flex) {
-      display: flex;
-      justify-content: space-between;
-      flex-wrap: nowrap;
-      align-items: center;
-    }
+  a {
+    line-height: 2;
```
Comment (U-DON) [inline src/sass/modules/_va-pagination.scss]: Since this got added, you can remove the `line-height: 1.5` from `&-inner`.

**Comment by U-DON** [comment]
LGTM; suggested some minor tweaks.

File: src/sass/modules/_va-pagination.scss (hunk 3 above)
Comment (webinista) [inline src/sass/modules/_va-pagination.scss]: That would also affect `va-pagination-inner`.

**Pull request merged**